Specs/HZFractionalMedia/1.5.1/HZFractionalMedia.podspec

```diff
@@ -0,0 +1,28 @@
+# To learn more about a Podspec see http://guides.cocoapods.org/syntax/podspec.html
+#
+
+Pod::Spec.new do |s|
+  s.name             = "HZFractionalMedia"
+  s.version          = "1.5.1"
+  s.summary          = "The Fractional Media SDK"
+  s.source           = { :http => "https://bintray.com/artifact/download/fyber/mobile-adapters-ios-tpn-sdk/adzone-ios-direct-v#{s.version}.zip" }
+  s.homepage         = "http://google.com"
```

Comment:
wouldn't `heyzap.com` or `fyber.com` be better?

Reply:
Yes it would. Lots of things could be changed in the pod spec, like the owner for example but I didn't want to introduce to much change. I leave these changes to the MOME team